conf/mesos-site.xml

```diff
@@ -38,18 +48,13 @@
   </property>
 
   <property>
-    <name>mesos.hdfs.executor.uri</name>
-    <value>http://path/to/executor</value>
-  </property>
-
-  <property>
     <name>mesos.hdfs.cluster.name</name>
     <value>hdfs-super-available</value>
```

Comment:
We could rename this, especially since we're not even "super-available" until we have scheduler HA working again.